about.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Audiowide&family=Exo+2:ital,wght@1,300;1,500;1,600&family=Open+Sans:wght@500&display=swap" rel="stylesheet">
+  <link href="stylesheets/style.css" rel="stylesheet"/>
+  <link href="stylesheets/about.css" rel="stylesheet"/>
+  <title>About Me</title>
+</head>
+
+
+
+<body>
+  <nav>
+    <ul class="nav">
+      <li class="link-1"><a class="nav-link" href="index.html">Main Page</a></li>
+      <li class="link-2"><a class="nav-link" href="portfolio.html">Projects</a></li>
+    </ul>
+  </nav>
+
+  <header class="header">
+    <h2>Background</h2>
+  </header>
+    <main>
+      <div class="back-color">
+        <h1>Interests</h1>
```

Comment:
Style: Your heading tags `h1`, `h2`, etc should allow you to construct an accurate table of contents.  

It looks like you used a `h1` here just to make the text larger, which is better accomplished with CSS.